### ISSUE TITLE
The documentation describing behaviour of the debug flag is obsolete

### ISSUE THREAD

**Issue by daveanderson-ie** [ckan/ckan#4454]
### CKAN Version if known (or site URL)
2.9.0a

### Please describe the expected behaviour
Documentation should describe the effect of setting the debug flag to true

### Please describe the actual behaviour
Documentation describes behaviour that was removed in https://github.com/ckan/ckan/pull/3000, according to this issue https://github.com/ckan/ckan/issues/3584

### What steps can be taken to reproduce the issue? 
1. Visit http://docs.ckan.org/en/latest/theming/templates.html
1. Review the documentation starting 'Now, let’s customize the CKAN front page'
1. Set the debug flag to true in the installation ini file (e.g. development.ini)
1. The debug link does not show the included templates

**Comment by wardi** [comment]
Thank you for this! Do you have a minute to submit a pull request to correct the documentation?

We're moving all the controllers over to flask views which IIUC will give us access to better, but different, interactive debugging. For now it's best to just remove this text.

**Comment by Gauravp-NEC** [comment]
@wardi , Please let me know if there is need to work on this issue now.

**Comment by Gauravp-NEC** [comment]
@wardi @amercader , Please let me know if there is need to work on this issue now. Thanks :)

**Comment by amercader** [comment]
@Gauravp-NEC yes, if you can review the steps on the documentation tutorial page linked to make sure they work with the latest CKAN version that would be great

**Comment by Gauravp-NEC** [comment]
@amercader , I have followed the document https://docs.ckan.org/en/latest/theming/templates.htmlhttps://docs.ckan.org/en/latest/theming/templates.html and configured the "debug= true" in my ckan.ini file after executing the command ckan -c /etc/ckan/default/ckan.ini run to run CKAN. I got the below error:

![Capture](https://user-images.githubusercontent.com/66116382/130436662-156451d8-4612-4ec9-a9c2-304397c04eab.JPG)
